File: 1264-maximum-number-of-words-you-can-type/1264-maximum-number-of-words-you-can-type.py
class Solution:
    def canBeTypedWords(self, text: str, brokenLetters: str) -> int:
        count = 0
        for word in text.split():
            flag = False

            for cw in word:
                for cbl in brokenLetters:
                    if (cw == cbl):
                        flag = True
                        break

                if flag:
                    break
            if not flag:
                count += 1

        return count
    
                

        

File: 1264-maximum-number-of-words-you-can-type/1264-maximum-number-of-words-you-can-type.cpp
class Solution {
public:
    int canBeTypedWords(string text, string brokenLetters) {
    std::istringstream iss(text);
    string word;
    int count = 0;

    // iterate through each word 
    while (iss >> word) {
        bool broken = false;

        for (char cw : word) {
            for (char cbl : brokenLetters) {
                if (cw == cbl) {
                    broken = true;  
                    break;         
                }
            }
            if (broken) {
                break;
            } 
        }
        if (!broken) {
            count++;
        }

    }
    
    return count;
    }

};